src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets

```diff
@@ -205,6 +207,39 @@
 
   </Target>
 
+  <Target Name="RestoreTextOnlyPackages">
+    <PropertyGroup>
+      <TextOnlyPackageRestoreDir>$(ArtifactsDir)text-only-packages/</TextOnlyPackageRestoreDir>
+      <TextOnlyPackageTarballDir>$(TarballRootDir)packages/text-only/</TextOnlyPackageTarballDir>
+    </PropertyGroup>
+
+    <Message Text="Restoring text only packages..." Importance="High" />
+
+    <MSBuild Projects="../Arcade/tools/TextOnlyPackages.csproj"
+             Properties="TargetPackagesPath=$(TextOnlyPackageRestoreDir)"
+             Targets="Restore" />
+
+    <ItemGroup>
+      <TextOnlyPackageContent
+        Include="
+          $(TextOnlyPackageRestoreDir)microsoft.dotnet.common.itemtemplates*/**/*;
```

Comment:
It would be nice if src/SourceBuild/Arcade/tools/TextOnlyPackages.csproj was the only touch point for adding/removing text only packages.  Can the include be a wildcard off of $(TextOnlyPackageRestoreDir)?

Reply:
Actually, restoring the project pulls in some other packages that aren't text-only:

```
microsoft.build.tasks.git/
microsoft.dotnet.xlifftasks/
microsoft.netcore.app.ref/
microsoft.sourcelink.azurerepos.git/
microsoft.sourcelink.common/
microsoft.sourcelink.github/
```

I thought about using a wildcard and explicitly excluding these. I decided against it because I didn't want to unintentionally include other packages in the future if this list changes. I also thought about returning the list of text only packages as output but couldn't get that working.

Reply:
I see that makes sense.  It is unfortunate there isn't a single touch point.

Reply:
Added text file on download of text-only packages to be used in copy so package names aren't duplicated.